packages/chakra-components/src/components/Election/Questions.tsx

```diff
@@ -280,14 +279,7 @@ const MultiChoice = ({ index, question }: QuestionProps) => {
   }
 
   const choices = [...question.choices]
-  if (election.resultsType.properties.canAbstain) {
-    choices.push({
-      title: {
-        default: localize('vote.abstain'),
-      },
-      value: -1,
-    })
-  }
+  const canAbstain = election.resultsType.properties.canAbstain
```

Comment:
Why define a variable if it's only used once? I guess you wanted to define it before, and use it in the condition above? 🤔

Reply:
84cd1a3